nin/backend/fontgen.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
+
+
+let fontGen = function(pathPrefix, cb) {
```

Comment:
I reccomend you switch fontGen to be async, so that we get away with one less indentation level in compile.js (soon our callbacks will be larger than the pyramids of Giza!).
you can switch fs.writeFileSync with await fs.outputFile from https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputFile.md.